pages/linux/xbps.md

```diff
@@ -0,0 +1,23 @@
+# xbps
+
+> The X Binary Package System (or xbps) is the binary package system used by Void Linux.
+
+- Install packages:
+
+`xbps-install -S {{package_name1}} {{package_name2}}`
+
+- Search for a package:
+
+`xbps-query -Rs {{package_name}}`
```

Comment:
Is there a full-form version of `-Rs` ? The word `R` doesn't exactly match with search. Hence my question.

Reply:
"The -R or --repository option switches to the repository mode. Most options are able to work in local and repository mode. "

and -s (not capital -S) stands for search

So, -Rs = Repository search

Reply:
Can we change to `-s --repository` and update the description to `Search for a package in repository mode` or something similar ?

Reply:
@agnivade you can use "xbps-query --repository --search {{package_name}}" sure. But it's very unlikely any user will be accustom to such syntax, since many tutorials and info online is always with the short (-Rs, -S etc.) modes and options.

But I can give some more information in the description if that helps. What do you think?

Reply:
@agnivade you can use "xbps-query --repository --search {{package_name}}" sure. But it's very unlikely any user will be accustom to such syntax, since many tutorials and info online is always with the short (-Rs, -S etc.) modes and options.

But I can give some more information in the description if that helps. What do you think?

Reply:
@isu17 - Our aim in tldr pages is more of an explainer than a cheatsheet -- in other words, the underlying mission is not to merely provide copy-pastable snippets that you can find in online tutorials, but rather to allow readers to understand the language of the commands, so that the logic behind the options is evident and they can then derive new commands from the provided examples. As you can see [here](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines) in pt 3. 

The way you have modified the descriptions is not something we do in our pages. We prefer free-flowing sentences that match their corresponding examples. I am still okay with `-s` rather than `--search` since the word search is there in the description. But let us go back to `--repository -s` instead of the short mode.

Reply:
Yeah, I get your point. I think I am starting to understand more and more what you mean, again sorry for the mishaps.

I will fix all the options and flags to be more descriptive so people would understand them easily.